### DISCUSSION TITLE
tips and tricks for adjusting the RPM trimmer?

### DISCUSSION THREAD

**Discussion by proteus075** [keirf/greaseweazle#124] [Q&A] [answered]
is it possible to "see" the rpm live through greaseweazle GUI or other way to be able to adjust the trimmer to 300 rpm on some floppy drive models?

**Comment by wohali** [comment, marked as answer]
https://cowlark.com/fluxengine/doc/greaseweazle.html supports the GW hardware and has the ability to determine disk rotation speed.

**Reply by Radiorama7** [reply]
I'm trying to do the same. I'm using fluxengine with a GW F7 plus.  I tried the following:
fluxengine read ibm -o ibm.img --usb.greaseweazle=COM4

And got:
Measuring rotational speed...
the drive LED lit up but the motor wouldn't run.

Maybe I'm using a drive that is set up as drive #1, so I added -s :d=1 (found in a forum):
>fluxengine read ibm -o ibm.img -s :d=1 --usb.greaseweazle=COM4

And obtained:
Error: unrecognised flux filename ':d=1'

What am I doing wrong?

Thanks!

**Reply by Radiorama7** [reply]
Ok, I found the correct syntax, I'll write it here for future reference:
-s drive:1
